src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue

```diff
@@ -145,5 +156,15 @@ export default {
 			}
 		}
 	},
+	mounted() {
```

Comment:
This could be done in `created` instead of `mounted` so that the banner would render on the server too